File: testjavabenchmark/domain/shortcharacter/identica/Message.java
package domain.shortcharacter.identica;

import infrastructure.asocialmedia.ASocialNetworkInfo;

//Functional requirement 3.1.1
//Test case 53: Class domain.shortcharacter.identica.Message may only have a dependency with class infrastructure.asocialmedia.ASocialNetwork
//Result: FALSE

public class Message{
	//FR5.5
	public String getNetworkInformation(ASocialNetworkInfo info){
		return "";
	}
}

File: testprojects/csharp/benchmark/domain/facebook/Message.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CSharpBenchmark.domain.facebook
{
    //Functional requirement 3.1.2
    //Test case 68: Only class presentation.gui.facebook.ChatGUI may have a dependency with domain.facebook.Message 
    //Result: FALSE
    public class Message
    {
        public static string getMessage()
        {
            return "chatmessage";
        }
    }
}
